test/verify/check-machines-dbus

```diff
@@ -226,6 +238,19 @@ class TestMachinesDBus(machineslib.TestMachines):
         b.wait_in_text("#pool-myPoolTwo-{0}-volume-VolumeOne-name".format(connectionName), "VolumeOne")
         b.wait_in_text("#pool-myPoolTwo-{0}-volume-VolumeTwo-name".format(connectionName), "VolumeTwo")
 
+        # Delete Storage Volume that is not used by any VM
+        b.click("tbody tr[data-row-id=pool-myPoolTwo-{0}-volume-VolumeTwo] input[type=checkbox]".format(connectionName))
+        b.click("#storage-volumes-delete")
+        b.wait_not_present("#storage-volumes-list tbody tr[data-row-id=pool-myPoolTwo-{0}-volume-VolumeTwo]".format(connectionName))
+
+        # Delete Storage Volume which is used by a VM
+        b.click("tbody tr[data-row-id=pool-myPoolTwo-{0}-volume-VolumeOne] input[type=checkbox]".format(connectionName))
+        b.click("#storage-volumes-delete")
+        b.wait_present("#storage-volumes-delete-modal")
+        b.wait_in_text("#storage-volumes-delete-modal ul", "VolumeOne is used by subVmTest1")
+        b.click("#storage-volumes-delete-modal .btn-danger")
+        b.wait_not_present("tbody tr[data-row-id=pool-myPoolTwo-{0}-volume-VolumeOne]".format(connectionName))
```

Comment:
I disallowed now the deletion of volumes that are used by VMs so that's not relevant anymore.

But just for the record, if the volume of a running VM gets deleted, the VM will still have the pointers to the inodes, so it will continue to work without issues. After shutting it down we 'll lose the disk.